### PR TITLE
Set `mqtt`-`clientId` to `uptime-kuma_..` instead of `mqttjs_..`

### PR DESCRIPTION
**Pull request by stheine**
…ker to identify where the connections are coming from

⚠️⚠️⚠️ Since we do not accept all types of pull requests and do not want to waste your time. Please be sure that you have read pull request rules:
https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md#can-i-create-a-pull-request-for-uptime-kuma

Tick the checkbox if you understand [x]:
- [x] I have read and understand the pull request rules.

# Description

I'm monitoring my MQTT broker for connections and found plenty that look like `mqttjs_466d1625`, so I wondered where they come from.
finally I found them to come from Uptime Kuma.

to make it easier to understand, I propose to set the MQTT `clientID` to include the term `uptime-kuma` to make it easy to identify.
the random part generation of the `clientID` follow the default format used by the `mqtt` lib, and I'm only setting a different prefix.
https://github.com/mqttjs/MQTT.js?tab=readme-ov-file#mqttclientstreambuilder-options

## Type of change

not sure which one you would pick for this kind of change, so I leave both active.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I ran ESLint and other linters for modified files
- [x] I have performed a self-review of my own code and tested it
- [ ] I have commented my code, particularly in hard-to-understand areas (including JSDoc for methods)
- [x] My changes generates no new warnings
- [ ] My code needed automated testing. I have added them (this is optional task)

## Screenshots (if any)

### CODE REVIEW

**Comment by CommanderStorm** [comment]
Would this break existing setups?

**Comment by stheine** [comment]
> Would this break existing setups?

I don't think it can, because the undefined `clientID` leads to the default from `mqtt`, and that same default is used in many other applications, so no one can rely on that default value coming from Uptime Kuma.

**Comment by CommanderStorm** [comment]
Let me rephrase:
Can changing the `clientID` change the monitoring outcome?
Can/Do people depend on this `clientID` being unchanged?

According to https://www.ibm.com/docs/en/ibm-mq/9.3?topic=concepts-client-identifier
this is used for servers to identify a client.
What happens if this is suddenly changed?

CC @tarun7singh

**Comment by stheine** [comment]
> Let me rephrase: Can changing the `clientID` change the monitoring outcome? Can/Do people depend on this `clientID` being unchanged?
> 
> According to https://www.ibm.com/docs/en/ibm-mq/9.3?topic=concepts-client-identifier this is used for servers to identify a client. What happens if this is suddenly changed?

the MQTT broker distributes the messages to all subscribed clients, no matter what their ID is. the distribution is based on the MQTT `topic`.
the `clientID` is just for identifying what clients register on the broker, but it does not affect what messages are sent to or from that client.

even today, it is suddenly changing, as the default already contains the random suffix.
the change of my PR is that the prefix of that random string is `uptime-kuma` instead of `mqttjs`.

**Review by CommanderStorm — APPROVED** [review]
Thanks for the confirmation ^^

**Comment by CommanderStorm** [comment]
Thanks for the bugfix!

All changes in this PR are small and uncontroversial ⇒ merging with junior maintainer approval
